packages/components/src/TimeFormatter/TimeFormatter.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+interface TimeFormatterProps {
+  /**
+   * Civil Time string to be formatted
+   * 17:45:00.000000000
+   * 11:30:00.000000000
+   */
+  readonly civilTime: string;
+
+  /**
+   * Decide to show 24 hour vs.
+   * 12 hour display
+   */
+  readonly useMilitaryTime: boolean;
+}
+
+export function TimeFormatter(props: TimeFormatterProps) {
+  const timeSplit = props.civilTime.split(":");
+  const hour = parseInt(timeSplit[0]);
+  const minute = parseInt(timeSplit[1]);
+
+  const amOrPm = getAmPmLabel(hour, props.useMilitaryTime);
+  const formattedHour = setCorrectHourFormat(hour, props.useMilitaryTime);
+
+  return `${formattedHour}:${minute} ${amOrPm}`;
+}
+
+function getAmPmLabel(hour: number, useMilitaryTime: boolean) {
+  if (useMilitaryTime === true) {
+    return "";
+  }
+
+  return hour >= 12 ? "PM" : "AM";
+}
+
+function setCorrectHourFormat(hour: number, useMilitaryTime: boolean) {
```

Comment:
@kkimble006 I noticed an edge case here: If you use `useMilitaryTime={false}` but set the hour to `0`, which is a valid hour in military time, it should return `12:xx AM`, but instead return `0:xx AM`, which is not a valid civilian time. 
![image](https://user-images.githubusercontent.com/12400201/62796338-3d6e2480-ba96-11e9-9597-1d913e99f486.png)

Reply:
Good Catch!